plover/registry.py

```diff
@@ -54,6 +55,17 @@ def update(self):
             entrypoint_type = 'plover.%s' % plugin_type
             for entrypoint in pkg_resources.iter_entry_points(entrypoint_type):
                 self.register_plugin(plugin_type, entrypoint)
+            if sys.platform.startswith('darwin'):
```

Comment:
Although we do use 'osx' elsewhere (`osx/README.md`, `osxkeyboardcontrol.py`, ...).